### DISCUSSION TITLE
Add support for pt_BR

### DISCUSSION THREAD

**Discussion by lucasjose501** [MohmmedAshraf/laravel-translations#5] [Ideas]
Hi! It looks like there is no support for pt_BR.
After installing the package and running `php artisan translations:import` this shows in the console:

`Language with code pt_BR not found`

My lang folder have `en` and `pt_BR` folders and in `app.php`config file the locale is set to `pt_BR` and `en` as fallback.

**Comment by MohmmedAshraf** [comment]
Hi @lucasjose501 ,

Thank you for letting me know about the issue with the pt_BR language.

I will make sure to add support for pt_BR in future releases of the package. I also plan to add support for more languages in future releases of the package.

Best regards

**Reply by lucasjose501** [reply]
Thank you @MohmmedAshraf 

I'll be watching for releases! Nice work

**Comment by MohmmedAshraf** [comment]
@lucasjose501 pt_BR #5  is now available in the latest release.
